### ISSUE TITLE
Exception handling - discussion 

### ISSUE THREAD

**Issue by Fahad-n** [rsanchez-wsu/RaiderPlanner#130]
We must agree on a way to handle the Exceptions in this project, so what do you think the best way to handle the Exceptions?

**Comment by Aashishkebab** [comment]
I think using try-catch is the best way to handle exceptions. That way, when an exception occurs, the application can handle it gracefully and accordingly.

**Comment by Bhulbert2096** [comment]
I agree with you @Aashishkebab that is better than just simply throwing an exception

**Comment by Fahad-n** [comment]
I agree with you too, because some part of the exceptions do use IF statements, throw exceptions or try- finally. 
So that is good we agreed on try-with-resources.

**Comment by rsanchez-wsu** [comment]
@Fahad-n @Aashishkebab @Bhulbert2096 I think that this discussion somewhat misses the point.  I think that @Fahad-n is trying to initiate a discussion based on what I mentioned in class about the need for an exception handling strategy.

The answer is more involved than just "use try-catch".  We need to reach a consensus on how best to log exceptions, what conventions will be used for the user-visible error messages, and which parts of the application will handle their own exceptions versus which parts will allow the exceptions to bubble up.
